pagination.py

```diff
@@ -0,0 +1,82 @@
+# -*- coding: utf-8 -*-
+"""
+    pagination.py
+
+    :copyright: (c) 2014 by Openlabs Technologies & Consulting (P) Limited
+    :license: BSD, see LICENSE for more details.
+"""
+from nereid.contrib.pagination import Pagination
+from trytond.pool import Pool
+from werkzeug.utils import cached_property
+
+
+class ElasticPagination(Pagination):
```

Comment:
Inherit from `BasePagination` please since you do not need the capabilities of ModelSQL pagination